source/MaterialXCore/Element.h

```diff
@@ -610,8 +608,8 @@ class MX_CORE_API Element : public std::enable_shared_from_this<Element>
     /// @param options Equivalence criteria
     /// @param results Results of comparison if argument is specified.
```

Comment:
Minor: The output argument is now `message` instead of results.

Reply:
Good catch, and I'll fix this.